x-pack/solutions/security/plugins/security_solution/public/common/store/reducer.ts

```diff
@@ -155,4 +160,5 @@ export const createReducer: (
     discover: securitySolutionDiscoverReducer,
     ...pluginsReducer,
     notes: notesReducer,
+    dataViewPicker: dataViewPickerReducer,
```

Comment:
nit. more generic naming?

Reply:
lets have a vote when we talk about it team wide then:)

Reply:
https://github.com/elastic/kibana/pull/210585/files#r1976190098 sounds okay actually